backend/fms_core/viewsets/study.py

```diff
@@ -13,4 +18,26 @@ class StudyViewSet(viewsets.ModelViewSet):
 
     filterset_fields = {
         **_study_filterset_fields,
-    }
\ No newline at end of file
+    }
+
+    def create(self, request, *args, **kwargs):
+        study_data = request.data
+
+        # Generate a sequential letter by counting the number of existing studies tied to the provided project
+        study_count = Study.objects.filter(project=study_data['project']).count()
```

Comment:
I would move these inside the create_study service unless we plan to someday allow non sequential studies (i doubt we'll ever do that).